src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Decompression.cs

```diff
@@ -306,5 +306,58 @@ public async Task GetAsync_SetAutomaticDecompression_HeadersRemoved(Configuratio
                 }
             });
         }
+
+        [Theory]
+#if NETCOREAPP
+        [InlineData(DecompressionMethods.GZip | DecompressionMethods.Deflate | DecompressionMethods.Brotli, "gzip; q=1.0, deflate; q=1.0, br; q=1.0")]
+#endif
+        [InlineData(DecompressionMethods.GZip | DecompressionMethods.Deflate, "gzip; q=1.0, deflate; q=1.0")]
```

Comment:
We should add a case here where the decompression handler will add at least one new header value, but not overwrite an existing one (or ones). 

e.g. something like `[InlineData(DecompressionMethods.GZip | DecompressionMethods.Deflate | DecompressionMethods.Brotli, "deflate; q=1.0")]`

and check that the result is "deflate; q=1.0, gzip, brotli".

Reply:
Added